### DISCUSSION TITLE
Can this project be used for object tracking in streaming video

### DISCUSSION THREAD

**Discussion by djhdj1** [FastTrackOrg/FastTrack#47] [Q&A]
I am a novice and want to make an object tracking of streaming video. Can this project help me? Thank you！

**Comment by bgallois** [comment]
FastTrack does not support direct tracking from a video stream natively.
It is possible to adapt FastTrack as mentioned at the end of the developer guide https://www.fasttrack.sh/API/ but it requires some coding.
